src/backend/InvenTree/plugin/plugin.py

```diff
@@ -239,9 +239,10 @@ def file(cls) -> Path:
         """File that contains plugin definition."""
         return Path(inspect.getfile(cls))
 
-    def path(self) -> Path:
+    @classmethod
+    def path(cls) -> Path:
         """Path to plugins base folder."""
-        return self.file().parent
+        return cls.file().parent
```

Comment:
Q: Is this related or just nice to have?

Reply:
This was needed for another approach I was trying - but in the end it was a nice addition